### ISSUE TITLE
listr -g option doesn't appear to recognize locally hosted refgenieserver genome names

### ISSUE THREAD

**Issue by jpsmith5** [databio/refgenie#135]
This may be a potential refgenieserver error, however, when attempting to restrict a `listr` command to a single genome with the `-g` option, it appears to not understand how to recognize a `Remote genomes: ` if that remote genomes is on a locally hosted refgenieserver.

```
$ refgenie listr -c test_multiple_servers.yaml -g hg19_cdna
Querying available assets from server: http://refgenomes.databio.org/v2/assets
Remote genomes: hg19_cdna
Remote assets:
           hg19_cdna/   fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, kallisto_index:default, salmon_index:default

Querying available assets from server: http://0.0.0.0:80/v2/assets
Remote genomes: test
Remote assets:
                test/   fasta.chrom_sizes:default, fasta.fai:default, fasta:default

Querying available assets from server: http://staging.refgenomes.databio.org/v2/assets
Remote genomes: hg19_cdna
Remote assets:
           hg19_cdna/   fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, kallisto_index:default, salmon_index:default
```

So, it's working to check multiple refgenieservers when listing assets. But, if I restrict by a genome present on the public servers (refgenomes.databio.org and staging...) it works, while still displaying the assets on the local host.  Conversely, if I try to restrict to the localhost genome ("test"), it instead displays all assets for all servers. See below:
```
$ refgenie listr -c test_multiple_servers.yaml -g test
Querying available assets from server: http://refgenomes.databio.org/v2/assets
Remote genomes: hg19, hg19_cdna, hg38, hg38_cdna, hg38_chr22, hg38_noalt_decoy, hs38d1, human_alphasat, human_alu, human_rDNA, human_repeats, mm10, mm10_cdna, mouse_chrM2x, rCRSd, rn6, rn6_cdna
Remote assets:
                hg19/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, ensembl_gtf.ensembl_gene_body:default, ensembl_gtf.ensembl_tss:default, ensembl_gtf:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, gencode_gtf:default, hisat2_index:default, refgene_anno.refgene_exon:default, refgene_anno.refgene_intron:default, refgene_anno.refgene_pre_mRNA:default, refgene_anno.refgene_tss:default, refgene_anno:default, star_index:default
           hg19_cdna/   fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, kallisto_index:default, salmon_index:default
                hg38/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, dbnsfp.tabix:default, dbnsfp:default, ensembl_gtf.ensembl_gene_body:default, ensembl_gtf.ensembl_tss:default, ensembl_gtf:default, ensembl_rb:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, feat_annotation:default, gencode_gtf:default, refgene_anno.refgene_exon:default, refgene_anno.refgene_intron:default, refgene_anno.refgene_pre_mRNA:default, refgene_anno.refgene_tss:default, refgene_anno:default
           hg38_cdna/   fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, kallisto_index:default, salmon_index:default
          hg38_chr22/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, star_index:default
    hg38_noalt_decoy/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, star_index:default
              hs38d1/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, star_index:default
      human_alphasat/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, star_index:default
           human_alu/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default
          human_rDNA/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, star_index:default
       human_repeats/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, star_index:default
                mm10/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, ensembl_gtf.ensembl_gene_body:default, ensembl_gtf.ensembl_tss:default, ensembl_gtf:default, ensembl_rb:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, feat_annotation:default, gencode_gtf:default, hisat2_index:default, refgene_anno.refgene_exon:default, refgene_anno.refgene_intron:default, refgene_anno.refgene_pre_mRNA:default, refgene_anno.refgene_tss:default, refgene_anno:default, star_index:default
           mm10_cdna/   fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, kallisto_index:default, salmon_index:default
        mouse_chrM2x/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, star_index:default
               rCRSd/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, star_index:default
                 rn6/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, refgene_anno.refgene_exon:default, refgene_anno.refgene_intron:default, refgene_anno.refgene_pre_mRNA:default, refgene_anno.refgene_tss:default, refgene_anno:default, star_index:default
            rn6_cdna/   fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, kallisto_index:default, salmon_index:default

Querying available assets from server: http://0.0.0.0:80/v2/assets
Remote genomes: test
Remote assets:
                test/   fasta.chrom_sizes:default, fasta.fai:default, fasta:default

Querying available assets from server: http://staging.refgenomes.databio.org/v2/assets
Remote genomes: hg19, hg19_cdna, hg38, hg38_cdna, hg38_chr22, hg38_noalt_decoy, hs38d1, human_alphasat, human_alu, human_rDNA, human_repeats, mm10, mm10_cdna, mouse_chrM2x, rCRSd, rn6, rn6_cdna
Remote assets:
                hg19/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, ensembl_gtf.ensembl_gene_body:default, ensembl_gtf.ensembl_tss:default, ensembl_gtf:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, gencode_gtf:default, hisat2_index:default, refgene_anno.refgene_exon:default, refgene_anno.refgene_intron:default, refgene_anno.refgene_pre_mRNA:default, refgene_anno.refgene_tss:default, refgene_anno:default, star_index:default
           hg19_cdna/   fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, kallisto_index:default, salmon_index:default
                hg38/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, dbnsfp.tabix:default, dbnsfp:default, ensembl_gtf.ensembl_gene_body:default, ensembl_gtf.ensembl_tss:default, ensembl_gtf:default, ensembl_rb:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, feat_annotation:default, gencode_gtf:default, refgene_anno.refgene_exon:default, refgene_anno.refgene_intron:default, refgene_anno.refgene_pre_mRNA:default, refgene_anno.refgene_tss:default, refgene_anno:default
           hg38_cdna/   fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, kallisto_index:default, salmon_index:default
          hg38_chr22/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, star_index:default
    hg38_noalt_decoy/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, star_index:default
              hs38d1/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, star_index:default
      human_alphasat/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, star_index:default
           human_alu/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default
          human_rDNA/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, star_index:default
       human_repeats/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, star_index:default
                mm10/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, ensembl_gtf.ensembl_gene_body:default, ensembl_gtf.ensembl_tss:default, ensembl_gtf:default, ensembl_rb:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, feat_annotation:default, gencode_gtf:default, hisat2_index:default, refgene_anno.refgene_exon:default, refgene_anno.refgene_intron:default, refgene_anno.refgene_pre_mRNA:default, refgene_anno.refgene_tss:default, refgene_anno:default, star_index:default
           mm10_cdna/   fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, kallisto_index:default, salmon_index:default
        mouse_chrM2x/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, star_index:default
               rCRSd/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, star_index:default
                 rn6/   bismark_bt1_index:default, bismark_bt2_index:default, bowtie2_index:default, bwa_index:default, fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, refgene_anno.refgene_exon:default, refgene_anno.refgene_intron:default, refgene_anno.refgene_pre_mRNA:default, refgene_anno.refgene_tss:default, refgene_anno:default, star_index:default
            rn6_cdna/   fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, kallisto_index:default, salmon_index:default

```

**Comment by michalstolarczyk** [comment]
ok, that's what it looks like on `dev` now:

```
[mstolarczyk@MichalsMBP test_genomes_new]: refgenie listr -c genomes.yaml -g hg19_cdna
Querying available assets from server: http://refgenomes.databio.org/v2/assets
Remote genomes: hg19_cdna
Remote assets:
           hg19_cdna/   fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, kallisto_index:default, salmon_index:default

Querying available assets from server: http://staging.refgenomes.databio.org/v2/assets
Remote genomes: hg19_cdna
Remote assets:
           hg19_cdna/   fasta.chrom_sizes:default, fasta.fai:default, fasta:default, hisat2_index:default, kallisto_index:default, salmon_index:default

Querying available assets from server: http://0.0.0.0:80/v2/assets
Genomes do not include: hg19_cdna
```



```
[mstolarczyk@MichalsMBP test_genomes_new]: refgenie listr -c genomes.yaml -g test
Querying available assets from server: http://refgenomes.databio.org/v2/assets
Genomes do not include: test
Querying available assets from server: http://staging.refgenomes.databio.org/v2/assets
Genomes do not include: test
Querying available assets from server: http://0.0.0.0:80/v2/assets
Remote genomes: test
Remote assets:
                test/   fasta.chrom_sizes:default, fasta.fai:default, fasta:default
```

**Comment by jpsmith5** [comment]
Great. Yes, that's looking perfect.
